packages/astro/src/integration/index.ts

```diff
@@ -0,0 +1,5 @@
+import { createIntegration } from './create-integration';
+
+export default createIntegration({
+  mode: 'hotload',
```

Comment:
is this overridable by the user? If yes, why?

Reply:
A developer cannot manipulate this code, or call `createIntegration` explictly.
what the can do though is import the clerk integration from `@clerk/astro/bunlded` instead of `@clerk/astro`

The default export of `@clerk/astro` uses the clerk integration with hotloading.